app/scripts/modules/core/src/search/global/globalSearch.directive.js

```diff
@@ -2,9 +2,10 @@
 
 const angular = require('angular');
 const $ = require('jquery');
+const SPINNER = require('../../widgets/spinners/spinner.component').SPINNER;
```

Comment:
This looks a little funky, could it potentially be moved to an import?

Reply:
Should definitely be an import, i.e.
```javascript
import { SPINNER } from 'core/widgets/spinnaker/spinner.component';
```

Reply:
I wanted to maintain the styling of the rest of the imports. I will update all of them, may be make it ts.

Reply:
Fixed.